### ISSUE TITLE
Allow editing of settings.toml

### ISSUE THREAD

**Issue by dgriswo** [circuitpython/web-editor#53]
It looks like there is a function to define all dot files as text.  I assume to accommodate the previous .env web workflow configuration. 

https://github.com/circuitpython/web-editor/blob/c36188587b5e4d46c985439676e86e2a879f4413/assets/js/common/file_dialog.js#L70

https://github.com/circuitpython/web-editor/blob/c36188587b5e4d46c985439676e86e2a879f4413/assets/js/common/file_dialog.js#L64-L66

Since .env files have been replaced with settings.toml, should the web editor have toml added to extensionMap and dot files removed?

**Comment by makermelissa** [comment]
Yes for sure on the .toml and probably yes on removing dot files.
